src/screens/RegisterConsent/RegisterConsentScreen.tsx

```diff
@@ -0,0 +1,92 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { useHeaderHeight } from '@react-navigation/elements';
+import React from 'react';
+import { View, Text, ScrollView, Linking } from 'react-native';
+import { RegisterStackParams } from '../../Navigator';
+import { fonts } from '../../util/fonts';
+import { spacing } from '../../util/spacing';
+import { colors } from '../../util/colors';
+import { CheckmarkRow } from './components/CheckMarkRow';
+import styles from './styles';
+import NextButton from '../../components/NextButton';
+
+const urlTermsOfService =
+  'https://docs.google.com/document/u/1/d/e/2PACX-1vQXqRRpBkB-7HqwLd2XtsWVDLjCUnBUIeNQADb56FuKHdj_IF9wbmsl4G7RLxR2_yKYMhnSO1M-X39H/pub';
+const urlPrivacyPolicy =
+  'https://docs.google.com/document/u/1/d/e/2PACX-1vTIL7a6NbUhBDxHmRy5tW0e5H4YoBWXUO1WvPseVuEATSLHMIemVAG6nnRe_xIJZ-s5YYPh2C05JwKR/pub';
+
+export type RegisterConsentScreenProps = NativeStackScreenProps<
+  RegisterStackParams,
+  'registerConsent'
+>;
+
+export const RegisterConsentScreen = (props: RegisterConsentScreenProps) => {
+  const headerHeight = useHeaderHeight();
+
+  const [selected1, setSelected1] = React.useState(false);
```

Comment:
Maybe we can define more understandable names.  
exp: 
selected1 -> permissionConfirmation
selected2 -> policyApproval

Reply:
Good point, but it was out of the scope of this task. For global refactoring we will have another task

Reply:
Note: this file was moved, not created by me